backend/aopp.go

```diff
@@ -268,6 +267,22 @@ func (backend *Backend) AOPPChooseAccount(code accounts.Code) {
 			return
 		}
 		signature = sig
+	case coinpkg.CodeETH:
+		sig, err := backend.keystore.SignETHMessage(
+			[]byte(backend.aopp.message),
+			addr.AbsoluteKeypath(),
+		)
+		if err != nil {
+			if firmware.IsErrorAbort(err) {
+				backend.log.WithError(err).Error("user aborted msg signing")
```

Comment:
you made a local `log` var on line 220 but continue to use `backend.log`. I think you meant to use the local `log` everywhere.